.github/workflows/tidy-modern.yml

```diff
@@ -0,0 +1,35 @@
+name: tidy-modern
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tidy:
+    runs-on: ubuntu-latest
+    container:
+      image: pointcloudlibrary/env:18.04
+    
+    steps:
+    - name: Install git
+      run: |
+          add-apt-repository ppa:git-core/ppa
```

Comment:
Any reason for installing git?

Reply:
I seem to remember that something in the Github Workflow had a dependency on git